google_bigquery_writer/app.py

```diff
@@ -1,45 +1,46 @@
-# coding=utf-8
 from keboola import docker
 from google_bigquery_writer.exceptions import UserException
 import google_bigquery_writer.writer
 import google.oauth2.credentials
+from google.cloud import bigquery
 import json
 from google_bigquery_writer import schema_mapper
+from google_bigquery_writer.bigquery_client_factory import BigqueryClientFactory
 
 
 class App:
-    def __init__(self, data_dir=None):
+    def __init__(self, data_dir: str = None):
```

Comment:
afaik to tady neni vubec potreba predavat, docker.Config by to mel nacist z env KBC_DATADIR

Reply:
Pri cistem behu zrejme ne. Ale predava se tim slozka s testovacimi daty. Napr. zde:
https://github.com/keboola/google-bigquery-writer/blob/d24040a2fd66b42e11e42936c54dbf57b539ba13/test/test_app.py#L79-L79

Reply:
no prave proto - pouziva se to jen v testech a tam by to taky melo jit pres env, to predavani pres commandlinu je takovy historicky pozustatek

Reply:
Akorat nikde v projektu neni definovana env `KBC_DATADIR`, ale to se tam asi dostava odjinud, ci?
Jestli tim historickym pozustatkem myslis to, jak se to spousti (viz niz) a ta env promenna co se tam predava je stejna, tak to bude OK.
https://github.com/keboola/google-bigquery-writer/blob/d24040a2fd66b42e11e42936c54dbf57b539ba13/Dockerfile#L20